main/isotp_ble_bridge_main.c

```diff
@@ -247,18 +331,17 @@ void app_main(void)
     ws2812_control_init(LED_GPIO_NUM);
     ws2812_write_leds(red_led_state);
 
-    // default to communicating with ECU
-    update_send_identifier(0x7E0);
-    update_receive_identifier(0x7E8);
-
     //Create semaphores and tasks
     websocket_send_queue = xQueueCreate(10, sizeof(send_message_t));
     tx_task_queue = xQueueCreate(10, sizeof(twai_message_t));
-    send_message_queue = xQueueCreate(10, sizeof(send_message_t));
-    isotp_task_sem = xSemaphoreCreateBinary();
+    isotp_send_message_queue = xQueueCreate(10, sizeof(send_message_t));
+//    isotp_task_sem = xSemaphoreCreateBinary();
```

Comment:
i also got rid of this entirely. if you are ok with it, let me know and i'll remove it. i don't like leaving behind commented out code for no good reason